__tests__/main.test.ts

```diff
@@ -19,6 +19,7 @@ beforeAll(() => {
 });
 test('Main', async () => {
   await run();
-  expect(octomock.mockFunctions.getInput).toHaveBeenCalledTimes(3);
+
+  expect(octomock.mockFunctions.getInput).toHaveBeenCalledTimes(6);
   expect(octomock.mockFunctions.createComment).toHaveBeenCalled();
```

Comment:
test fails here

Reply:
I just packaged octomock into an npm package and added some breaking changes with a refactor.  I bet that is part of the issue.  Let me try to pin this to `v0.0.1`